deploy/kubernetes/manifest.yaml

```diff
@@ -63,6 +75,15 @@ spec:
               mountPath: /csi
             - name: registration-dir
               mountPath: /registration
+        - name: liveness-probe
+          imagePullPolicy: Always
+          image: quay.io/k8scsi/livenessprobe:v1.1.0
+          args:
+            - --csi-address=/csi/csi.sock
+            - --health-port=9809
```

Comment:
The default [probe-timeout](https://github.com/kubernetes-csi/livenessprobe/blob/v1.1.0/cmd/livenessprobe/main.go#L37) is 1 sec. Have you tested if this is long enough?

Reply:
yes, had no issue. I think socket taking >1s to respond when they are in the same pod would be worrisome.